ApiBundle/Transformer/ContentTransformer.php

```diff
@@ -75,7 +75,8 @@ public function transform($content)
             $facade->addAttribute($contentAttribute);
         }
 
-        if ($this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_UPDATE_CONTENT_TYPE_FOR_CONTENT)) {
+        if ($this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_CONTENT_TYPE_FOR_CONTENT)
+            && $this->authorizationChecker->isGranted(ContentTypeForContentPanelStrategy::ROLE_ACCESS_UPDATE_CONTENT_TYPE_FOR_CONTENT)) {
```

Comment:
Why check update AND access to edit ? I think it's the only location where we're doing this isn't it ? The ROLE_ACCESS_UPDATE_CONTENT_TYPE_FOR_CONTENT allows edition even if the user doen't have ROLE_ACCESS_CONTENT_TYPE_FOR_CONTENT no ?